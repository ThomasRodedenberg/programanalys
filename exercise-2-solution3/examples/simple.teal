fun main() = {
    var x : nonnull int := 5; // N 2 27 =, N 2 4 +
    var y : nonnull int := null; // N 3 27 =, N 3 4 0
    var z := 3; // N 4 13 =, N 4 4 =
    x := y; // N 5 4 0
    z := x; // N 6 4 =
    // N 0 0 =
}