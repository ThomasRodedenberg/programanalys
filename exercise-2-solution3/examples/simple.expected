N 2 4 +
N 3 4 0
N 5 4 0
